File: CS/NetCore/WhatIsMyBrowser.CommonTypesCore/SearchMetaData.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace WhatIsMyBrowser.CommonTypesCore
{
    public class SearchMetaData
    {
        [JsonProperty("num_of_results_returned")]
        public long NumOfResultsReturned { get; set; }

        [JsonProperty("search_took_milliseconds")]
        public long SearchTookMilliseconds { get; set; }

        [JsonProperty("search_parameters")]
        public Dictionary<string, object> SearchParameters { get; set; }
    }
}

File: VB/NET40/WhatIsMyBrowser.CommonTypesVB/SearchMetaData.vb
Imports Newtonsoft.Json

Public Class SearchMetaData
    <JsonProperty("num_of_results_returned")>
    Public Property NumOfResultsReturned As Long

    <JsonProperty("search_took_milliseconds")>
    Public Property SearchTookMilliseconds As Long

    <JsonProperty("search_parameters")>
    Public Property SearchParameters As Dictionary(Of String, Object)
End Class